OSVR-Unity/CHANGES.md

```diff
@@ -5,6 +5,15 @@ This is an abbreviated changelog for the OSVR Unity Plugin.
 Use git for a full changelog.
 ##Recent Changes
 
+### InterfaceAdapter update
+> 07-July-2015 - v0.2.23 (commit c3c5dc0) and approximately v0.2.158-gb66f7af
+
+- InterfaceGameObject is still available, but has been deprecated. Use OrientationInterface, PoseInterface, etc... instead.
```

Comment:
For this and other mentions of code, can you mark them up with markdown backticks so they show up as code?